vsintegration/src/FSharp.Editor/FSharp.Editor.resx

```diff
@@ -212,7 +212,8 @@ Time until stale results are used (in milliseconds);
 Parallelization (requires restart);
 Enable parallel type checking with signature files;
 Enable parallel reference resolution;
-Enable fast find references &amp; rename (restart required)</value>
+Enable fast find references &amp; rename (restart required);
+Cache typechecking results (experimental)</value>
```

Comment:
This misleads into thinking this wasn't done already in general.
The name should either reflect it, or maybe this isn't really important to be a VS feature?

(it should work for plain SDK compilations as well, so a project setting would make more sense here)

Reply:
Removed from VS now.